Mage.Sets/src/mage/cards/s/SlivdraziMonstrosity.java

```diff
@@ -0,0 +1,109 @@
+        this.subtype.add(SubType.SLIVER);
+        this.subtype.add(SubType.ELDRAZI);
+        this.power = new MageInt(8);
+        this.toughness = new MageInt(8);
+
+        // Eldrazi you control are Slivers in addition to their other types.
+        this.addAbility(new SimpleStaticAbility(new BecomesSubtypeAllEffect(
+                Duration.WhileOnBattlefield, Arrays.asList(SubType.SLIVER), eldrazi_you_control, false
+        ).setText("Eldrazi you control are Slivers in addition to their other types.")));
+
+        // Slivers you control have devoid and annihilator 1.
+        Ability ability = new SimpleStaticAbility(new SlivdraziMonstrosityEffect());
+        ability.addEffect(
+                new GainAbilityControlledEffect(
+                new AnnihilatorAbility(1), Duration.WhileOnBattlefield,
+                StaticFilters.FILTER_PERMANENT_SLIVERS
+        ).setText("and annihilator 1"));
+        this.addAbility(ability);
+
+        // {3}: Create a 1/1 colorless Eldrazi Sliver creature token. It has “Sacrifice this creature: Add {C}.”
+        this.addAbility(new SimpleActivatedAbility(Zone.BATTLEFIELD, new CreateTokenEffect(new EldraziSliverToken()), new ManaCostsImpl<>("{3}")));
+    }
+
+    private SlivdraziMonstrosity(final SlivdraziMonstrosity card) {
+        super(card);
+    }
+
+    @Override
+    public SlivdraziMonstrosity copy() {
+        return new SlivdraziMonstrosity(this);
+    }
+}
+
+class SlivdraziMonstrosityEffect extends ContinuousEffectImpl {
+
+    private static final FilterPermanent filter = new FilterPermanent(SubType.SLIVER, "Slivers");
+
+    private static ObjectColor colorless = new ObjectColor();
+
+    public SlivdraziMonstrosityEffect() {
+        super(Duration.WhileOnBattlefield, Layer.ColorChangingEffects_5, SubLayer.NA, Outcome.Benefit);
+        staticText = "Slivers you control have devoid";
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        for (Permanent perm : game.getBattlefield().getActivePermanents(filter, source.getControllerId(), source, game)) {
+            if (perm.isControlledBy(source.getControllerId())) {
+                // Due to the carefully calibrated system of layers that Magic uses to determine the interaction of continuous
+                // effects, gaining an ability that changes the color of an object has no effect on the color of that object.
+                // Despite this, Slivdrazi Monstrosity causes Slivers to become colorless as they gain devoid. Rather than try
+                // to figure out how this could work, this card should just be played as though it read “Slivers you control
+                // have devoid and annihilator 1 and are colorless.”
+                // (2019-11-12)
+                // This ruling implemented here by adding this ability in the color-changing effect layer.
+                perm.addAbility(new DevoidAbility(perm.getColor(game)), source.getSourceId(), game);
```

Comment:
Have you tested that the slivers become colorless and that they properly interact with things that only work on colorless objects (or that they no longer work with ones that target only colored permanents)?

Reply:
Confirmed that Ghostfire Blade can equip a normally-colored sliver for 1, rather than 3, and that Ugin's roll-down doesn't exile Slivdrazi Monstrosity or other slivers.